src/Apps/Legion/Models/Interfaces.ts

```diff
@@ -55,14 +55,19 @@ export interface IAppData {
 
 /** to be used for Database ingestion flow */
 export interface IADXConnection {
+    selectedSourceCluster: string;
     selectedSourceDatabase: string;
     selectedSourceTable: string;
     selectedSourceTwinIDColumn: string;
     selectedSourceTableType: string;
-    selectedTargetDatabase: string;
 }
 
 export interface IPIDDocument {
-    url: string;
-    name: string;
+    pidUrl: string;
+}
+
+export interface IReactSelectOption {
```

Comment:
Interfaces used to be more powerful since it was letting extends. But now we can also extend types via intersections with the ts version we use today. 
Only power of interfaces is being able to add additional properties to existing interface. https://www.typescriptlang.org/docs/handbook/2/everyday-types.html#differences-between-type-aliases-and-interfaces

Since we will not need additional properties to extend the existing interface, we can keep it in types then.

Reply:
I will move my definition to Types and you can resolve conflict in that file if happens

Reply:
oh i see. we were just separating them based on if they were a `Type` vs `interface`? I didn't realize that. Personally I would rather break them up by how they are used or what they are for.